exporter/otlphttpexporter/README.md

```diff
@@ -30,9 +30,6 @@ The following settings can be optionally configured:
   - `ca_file` path to the CA cert. For a client this verifies the server certificate. Should only be used if `insecure` is set to false.
   - `cert_file` path to the TLS cert to use for TLS required connections. Should only be used if `insecure` is set to false.
   - `key_file` path to the TLS key to use for TLS required connections. Should only be used if `insecure` is set to false.
-
-- `compression` (default = none): Compression type to use (only gzip is supported today)
```

Comment:
I think this can stay in the documentation since it can still be set, but will apply to the HTTPConfig settings.